lib/ui/nav_pages/dm_page/dm_page.dart

```diff
@@ -49,47 +48,30 @@ class DmPage extends StatelessWidget {
                       left: 6,
                       right: 6),
                   const SizedBox(height: 30),
-                  GestureDetector(
-                    onTap: () => model.navigateToDmUserView(),
-                    child: const DMMessageUnread(),
-                  ),
-                  const SizedBox(height: 20),
-                  GestureDetector(
-                    onTap: () => model.navigateToDmUserView(),
-                    child: const DMMessageRead(),
-                  ),
-                  const SizedBox(height: 20),
-                  const DMMessageUnread(),
-                  const SizedBox(height: 20),
-                  GestureDetector(
-                    onTap: () => model.navigateToDmUserView(),
-                    child: const DMMessageRead(),
-                  ),
-                  const SizedBox(height: 20),
-                  GestureDetector(
-                    onTap: () => model.navigateToDmUserView(),
-                    child: const DMMessageRead(),
-                  ),
-                  const SizedBox(height: 20),
-                  GestureDetector(
-                    onTap: () => model.navigateToDmUserView(),
-                    child: const DMMessageRead(),
-                  ),
-                  const SizedBox(height: 20),
-                  GestureDetector(
-                    onTap: () => model.navigateToDmUserView(),
-                    child: const DMMessageRead(),
-                  ),
-                  const SizedBox(height: 20),
-                  GestureDetector(
-                    onTap: () => model.navigateToDmUserView(),
-                    child: const DMMessageRead(),
-                  ),
-                  const SizedBox(height: 20),
-                  GestureDetector(
-                    onTap: () => model.navigateToDmUserView(),
-                    child: const DMMessageRead(),
-                  ),
+                  StreamBuilder<List<DmListItem>>(
```

Comment:
Your stream should be in the view model. And the ViewModel should be StreamViewModel and then you will have an override function where all your streams will be stored.